be/src/storage/rowset/column_reader.h

```diff
@@ -102,7 +102,7 @@ class ColumnReader {
     void operator=(ColumnReader&&) = delete;
 
     // create a new column iterator. Caller should free the returned iterator after unused.
-    StatusOr<std::unique_ptr<ColumnIterator>> new_iterator();
+    StatusOr<std::unique_ptr<ColumnIterator>> new_iterator(ColumnAccessPath* path = nullptr);
```

Comment:
maybe add a new method for map and struct type is better?

Reply:
No, it's recursive call...